commessa/src/main/java/com/example/commessa/model/Commessa.java

```diff
@@ -9,8 +9,11 @@
 import jakarta.persistence.Temporal;
 import jakarta.persistence.TemporalType;
 import java.util.Date;
+import lombok.Getter;
```

Comment:
Perché usare esplicitamente Getter e Setter e non un più generico Data?
Le entità dovrebbero avere anche il metodo Hash, se vuoi avere la possibilità di gestirle in una HashMap, un costruttore senza parametri (perché serve a Hibernate) uno con i parametri perché è utile durante lo sviluppo; un metodo ToString, per poterle "visualizzare" nei log, un metodo Equal per verificare se due entità sono la stessa al di la della sola posizione di memoria, ...